src/server/dflycmd.cc

```diff
@@ -120,6 +120,18 @@ void DflyCmd::Run(CmdArgList args, ConnectionContext* cntx) {
     return ReplicaOffset(args, cntx);
   }
 
+  if (sub_cmd == "CLUSTER" && args.size() > 2) {
+    if (!ClusterConfig::IsClusterEnabled()) {
+      return (*cntx)->SendError("DFLY CLUSTER commands requires --cluster_mode=yes");
+    }
+    ToUpper(&args[1]);
+    string_view sub_cmd = ArgS(args, 1);
```

Comment:
Perhaps it'd be better then to wrap this in a function, like other sub commands in this file. We also know for a fact that other sub commands of `CLUSTER` will be added, so it's going to be like that anyway.
If you do wrap it in a function, we can use `sub_cmd` as it will no longer hide the other var.